catalog/spatial/kml/spatial-kml-util/src/main/java/org/codice/ddf/spatial/kml/converter/MetacardToKml.java

```diff
@@ -111,16 +115,22 @@ private static Geometry createPointGeo(Point jtsPoint) {
     return KmlFactory.createPoint().addToCoordinates(jtsPoint.getX(), jtsPoint.getY());
   }
 
-  private static com.vividsolutions.jts.geom.Geometry readGeoFromWkt(final String wkt)
+  public static com.vividsolutions.jts.geom.Geometry getJtsGeoFromWkt(final String wkt)
       throws CatalogTransformerException {
+
+    if (StringUtils.isBlank(wkt)) {
+      throw new CatalogTransformerException(
+          "WKT was null or empty. Unable to preform KML Transform on Metacard.");
```

Comment:
This probably needs to change to "WKT was null or empty. Unable to convert WKT to Jts Geometry."